integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -287,21 +273,19 @@ def export(self, **kwargs):  # pylint: disable=R0915
             enterprise_customer_identifier=self.enterprise_customer.name
         )
 
-        for enterprise_enrollment in enrollment_queryset:
-            is_audit_enrollment = enterprise_enrollment.is_audit_enrollment
-            if TransmissionAudit and skip_transmitted and \
-                    is_already_transmitted(TransmissionAudit, enterprise_enrollment.id, grade):
-                # We've already sent a completion status for this enrollment
-                generate_formatted_log(
-                    'Skipping export of previously sent enterprise enrollment. '
-                    'EnterpriseEnrollment: {enterprise_enrollment_id}'.format(
-                        enterprise_enrollment_id=enterprise_enrollment.id
-                    ),
-                    channel_name=channel_name,
-                    enterprise_customer_identifier=self.enterprise_customer.name
-                )
-                continue
+        if TransmissionAudit and skip_transmitted:
```

Comment:
This refactor seems logical, does it pass any existing tests (are there existing tests for the export() function?)

Reply:
yup it passes existing tests (in fact I had to edit existing tests to ensure they passes as they no longer can use the mocks for the api clients